cpp/NativeOpacityTurboModule.cpp

```diff
@@ -3,21 +3,24 @@
 #include "opacity.h"
 #include <thread>
 
-namespace facebook::react {
-
-NativeOpacityTurboModule::NativeOpacityTurboModule(
-    std::shared_ptr<CallInvoker> jsinvoker)
-    : NativeOpacityCxxSpec(std::move(jsinvoker)) {}
-
-jsi::Value NativeOpacityTurboModule::init(jsi::Runtime &rt, std::string api_key,
-                                          bool dry_run, double environment) {
-  jsi::Function promiseConstructor =
-      rt.global().getPropertyAsFunction(rt, "Promise");
+namespace facebook::react
+{
+
+  NativeOpacityTurboModule::NativeOpacityTurboModule(
```

Comment:
Hmmm what's up with all the indentation changes. Can you undo this? I'm using VSCode to format the code with the clang-format extension